components/ToDo/index.js

```diff
@@ -0,0 +1,55 @@
+import useLocalStorageState from "use-local-storage-state";
+import {
+  InputField,
+  ToDoBody,
+  AddButton,
+  DeleteButtonToDo,
+  ToDoBackground,
+  ToDoP,
+  InputPosition,
+} from "./index.styled";
+
+export default function ToDo() {
+  const [todos, setTodos] = useLocalStorageState("todos", { defaultValue: [] });
+  const [task, setTask] = useLocalStorageState("task", { defaultValue: "" });
```

Comment:
You don't need to use local storage for the controlled input of the task. `useState` is enough.